src/test/webServer.ts

```diff
@@ -33,19 +33,19 @@ const newProcessLogPrefixer = () => new stream.Transform({
   },
 });
 
-class LaunchServer {
+export class WebServer {
```

Comment:
This doesn't need to be exported right?
```suggestion
class WebServer {
```